src/Bicep.LangServer/Handlers/BicepDocumentSymbolHandler.cs

```diff
@@ -64,6 +64,9 @@ private SymbolKind SelectSymbolKind(DeclaredSymbol symbol)
         {
             switch (symbol)
             {
+                case ImportedNamespaceSymbol _:
+                    return SymbolKind.Package;
```

Comment:
>Package [](http://example.com/codeflow?start=38&length=7)

Btw, there's also `Namespace` in that enum. Any reason why we're not using that one? Is it because the icon is the same as object?

Reply:
I scanned the list, but apparently not hard enough 😉. Updated to use Namespace.